src/common/modals/JavaSetup.js

```diff
@@ -63,25 +68,55 @@ const JavaSetup = () => {
                 & > div {
                   display: flex;
                   justify-content: center;
-                  margin-top: 30px;
                 }
               `}
             >
-              <div>
-                <Button
-                  type="primary"
-                  css={`
-                    width: 150px;
-                  `}
-                  onClick={() => {
-                    setStep(1);
-                    setChoice(0);
-                  }}
-                >
-                  Automatic Setup
-                </Button>
+              <div
+                css={`
+                  display: flex;
+                  align-items: center;
+                  margin-top: 30px;
+                `}
+              >
+                <div>
+                  <Button
+                    type="primary"
+                    css={`
+                      width: 150px;
+                      margin-right: 10px;
+                    `}
+                    onClick={() => {
+                      setStep(1);
+                      setChoice(0);
+                      setForce(false);
+                    }}
+                  >
+                    Automatic Setup
+                  </Button>
+                </div>
+                or
+                <div>
+                  <Button
+                    type="default"
+                    css={`
+                      width: 150px;
+                      margin-left: 10px;
+                    `}
+                    onClick={() => {
+                      setStep(1);
+                      setChoice(0);
+                      setForce(true);
+                    }}
+                  >
+                    Force installation
```

Comment:
Why force installation? What's the purpose of this?

Reply:
That’s from me. The current method only updates the versions which are either completely missing or when the executable don’t works. The problem here is that if you mess whit the filesystem inside the Java version, the executable could pass that test but still won’t work for Minecraft. In that case you could click the Button „Force installation“ to fully re-install both versions.

Reply:
This shouldn't be on the user, when the user does the automatic installation you can also check if java actually works by running a standard java command and waiting for the correct output. If it fails you force install it anyway

Reply:
We do that but just -version

Reply:
So shall I remove it? It was thought to be a feature to make our life easier so if someone stumbles over issues whit Java we can just say „ah did you try to fully reinstall it?“ and I think all the Java problems should be resolved whit doing that.

Reply:
Yes, remove it, please. I just noticed we already validate the java installation by running a `-version`, that's enough. If java didn't download correctly or doesn't work it will trigger a forced download anyway even in the automatic setup.